### PR TITLE
Upgrade SQLite on both PHP 8.2 and 8.3

### PR DESCRIPTION
**Pull request by GrahamCampbell**

### CODE REVIEW

File: php-84/Dockerfile [hunk 1]
```diff
@@ -378,7 +378,8 @@ RUN make && make install
 # SQLite
 # Since PHP 7.4, libsqlite must be installed (https://github.com/php/php-src/blob/99b8e67615159fc600a615e1e97f2d1cf18f14cb/UPGRADING#L616-L619)
 # PHP 8.4 requires SQLite 3.13.0 or higher because the build uses the
-# `SQLITE_DBCONFIG_ENABLE_LOAD_EXTENSION` constant. The system version is 3.7.17.
+# `SQLITE_DBCONFIG_ENABLE_LOAD_EXTENSION` constant. Moreover, Laravel 11
+# requires SQLite 3.35.0 or higher. The system version is 3.7.17.
```
Comment (mnapoli) [inline php-84/Dockerfile:382]: Do you have a link on why Laravel requires a specific SQLite version?

Comment (GrahamCampbell) [reply php-84/Dockerfile:382]: Laravel 11 also requires PHP 8.2 or higher, so it's not necessarily to backport this change to PHP 8.1

Comment (GrahamCampbell) [reply php-84/Dockerfile:382]: https://github.com/laravel/framework/pull/48864 was the change that drove this btw.

Comment (mnapoli) [reply php-84/Dockerfile:382]: Thanks, I updated the documentation in https://github.com/brefphp/aws-lambda-layers/pull/189

Comment (hafezdivandari) [reply php-84/Dockerfile:382]: @GrahamCampbell we have extended SQLite support to 3.26+ on Laravel 11.15: laravel/docs#9768

Comment (mnapoli) [reply php-84/Dockerfile:382]: Awesome, thanks for letting us know! The system version is 3.7 so we still need to compile sqlite manually, but it's good to know!

**Pull request merged**